tools/ci_connector_ops/ci_connector_ops/pipelines/builds/normalization.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from typing import Tuple
+
+from ci_connector_ops.pipelines.actions import environments
+from ci_connector_ops.pipelines.bases import Step, StepResult, StepStatus
+from ci_connector_ops.pipelines.contexts import ConnectorContext
+from ci_connector_ops.utils import Connector
+from dagger import Container
+
+
+class BuildOrPullNormalization(Step):
```

Comment:
Hey @alafanechere Am I right that this is currently not in use? or am I blind?

Reply:
It is used, but [only in the test context for java destination](https://github.com/airbytehq/airbyte/blob/b98cf519f3c95bcc42e19b749430baac5f2e550e/tools/ci_connector_ops/ci_connector_ops/pipelines/tests/java_connectors.py#L80)